source/animate.css

```diff
@@ -78,8 +78,10 @@
 @import 'flippers/flipOutY.css';
 
 /* Lightspeed */
-@import 'lightspeed/lightSpeedIn.css';
-@import 'lightspeed/lightSpeedOut.css';
+@import 'lightspeed/lightSpeedInRight.css';
```

Comment:
Demo or it didn't happen :P

Reply:
Here's the demo @eltonmesquita https://codepen.io/warengonzagaofficial/full/mdJayJG